client/src/app/shared/constants.ts

```diff
@@ -0,0 +1 @@
+export const INVALID_FILE_TYPES = ['Invalid Metadata', 'Invalid File', 'Not Uploaded'];
```

Comment:
I think we need to distinguish between 2 error types only - MetadataError and FileError. I'd categorise not uploaded as a file error instead of different type. File not uploaded can be checked independently of the validationErrors for e.g. by checking the cloudUrl is set or not.

Reply:
We need all three in the UI right? The user needs to be able to filter between the three.

Reply:
Yea, I wasn't seeing file not uploaded as necessarily an error but i've included it as an errortype in the validator now and the repository queries in core are also being updated to reflect that.

Reply:
So is this okay to be merged?

Reply:
Looks good but I'd get someone more familiar with the UI to have a look as well.